### PR TITLE
[NUI] Change the values of some enumeration types of InputMethod

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- In Tizen.NUI.InputMethod.ActionButtonTitleType, some enumeration types
 are not used anymore : Previous, Unspecified, None
- To match the types of ecore_imf, some values should be changed.

### API Changes ###
- TCSACR-379
https://code.sec.samsung.net/jira/browse/TCSACR-379

### CODE REVIEW

**Review by AdunFang — APPROVED** [review]
It is OK.

**Comment by Seoyeon2Kim** [comment]
In dali module, these enumeration types are removed now because they're unused anymore and don't exist in ecore_imf.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-adaptor/+/244652/
I'll request a review later.

File: src/Tizen.NUI/src/public/InputMethod.cs [hunk 1]
```diff
@@ -387,7 +391,14 @@ private PropertyMap ComposingInputMethodMap()
...
             PropertyMap _outputMap = new PropertyMap();
             if (_panelLayout != null) { _outputMap.Add("PANEL_LAYOUT", new PropertyValue((int)_panelLayout)); }
-            if (_actionButton != null) { _outputMap.Add("BUTTON_ACTION", new PropertyValue((int)_actionButton)); }
+            if (_actionButton != null)
+            {
+                // Temporarily specify the values to match the types of ecore_imf.
+                if (_actionButton == InputMethod.ActionButtonTitleType.Search) _actionButton = (InputMethod.ActionButtonTitleType.Search - 1); // 6
+                else if (_actionButton == InputMethod.ActionButtonTitleType.Send) _actionButton = (InputMethod.ActionButtonTitleType.Send - 1); // 7
+                else if (_actionButton == InputMethod.ActionButtonTitleType.SignIn) _actionButton = (InputMethod.ActionButtonTitleType.SignIn - 1); // 8
+                _outputMap.Add("BUTTON_ACTION", new PropertyValue((int)_actionButton));
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/InputMethod.cs:400]: Is it unnecessary to add an exception for the Unspecified/None??

Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/InputMethod.cs:400]: Yes, you're right.
I'll update the patch for these exceptions.

**Review by huiyueun — APPROVED** [review]

**Pull request merged**